forum/qa-plugin/outdated-question-info/frontend/show-outdated-question-info.js

```diff
@@ -7,23 +7,28 @@ if(placeOfOutdatedQuestionInfo){
     const publishMonthNewerThanNow = publishDate.getMonth() - 1 >= now.getMonth();
 
     if (publishYearOlderThanNow && publishMonthNewerThanNow) {
-        let infoDisplay = "";
-        if(placeOfOutdatedQuestionInfo.style.display !== 'none'){
-            infoDisplay = "display:block;";
-        }else{
-            infoDisplay = "display:none;";
+        let shouldBeDisplayed = "";
+        if(placeOfOutdatedQuestionInfo.style.display === 'none'){
+            shouldBeDisplayed = "hidden";
         }
         placeOfOutdatedQuestionInfo.insertAdjacentHTML('beforebegin', 
-                `<p class = "qa-outdated-question-container" style = "${infoDisplay}">
+                `<p class = "qa-outdated-question-container ${shouldBeDisplayed}">
                     To pytanie zostało zadane już dawno temu i może być nieaktualne.<br/>
                     Upewnij się, że Twoja odpowiedź nadal będzie pomocna.
                 </p>`);
     }
     const QuestionElemExist = document.querySelector('.qa-outdated-question-container');
```

Comment:
Dużą literą powinno nazywać się klasy lub funkcje-konstruktory, a DOM element to jest zwykły obiekt, więc lepiej jest użyć *camelCase*.